be/src/storage/storage_engine.cpp

```diff
@@ -109,6 +109,23 @@ void StorageEngine::load_data_dirs(const std::vector<DataDir*>& data_dirs) {
     threads.reserve(data_dirs.size());
     for (auto data_dir : data_dirs) {
         threads.emplace_back([data_dir] {
+            uint64_t live_sst_files_size_before = 0;
```

Comment:
I have one question, if we add this feature, compaction will take a long time. In this case, the restarting Backend can't provide service. it may cause ability problem.
I suggest we should introduce a config to switch this feature.
Users can open it when they need this feature.

Reply:
Good idea, changed.